memory/src/main/java/org/opensearch/ml/memory/action/conversation/CreateInteractionTransportAction.java

```diff
@@ -93,10 +96,9 @@ protected void doExecute(Task task, CreateInteractionRequest request, ActionList
         Integer traceNumber = request.getTraceNumber();
         try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().newStoredContext(true)) {
             ActionListener<CreateInteractionResponse> internalListener = ActionListener.runBefore(actionListener, () -> context.restore());
-            ActionListener<String> al = ActionListener
-                .wrap(iid -> { internalListener.onResponse(new CreateInteractionResponse(iid)); }, e -> {
-                    internalListener.onFailure(e);
-                });
+            ActionListener<String> al = ActionListener.wrap(iid -> {
+                cmHandler.updateConversation(cid, new HashMap<>(), getUpdateResponseListener(cid, iid, internalListener));
+            }, e -> { internalListener.onFailure(e); });
             if (parintIid == null || traceNumber == null) {
                 cmHandler.createInteraction(cid, inp, prompt, rsp, ogn, additionalInfo, al);
```

Comment:
[nit] `cid`, `inp`, `rsp`, `ogn` is super hard to understand.

Reply:
can update in a separate PR. These names are consistently used in the repo.